pkg/whitesource/scanNPM.go

```diff
@@ -80,6 +81,8 @@ func (s *Scan) writeWhitesourceConfigJSON(config *ScanOptions, utils Utils, devD
 
 // ExecuteNpmScan iterates over all found npm modules and performs a scan in each one.
 func (s *Scan) ExecuteNpmScan(config *ScanOptions, utils Utils) error {
+	s.AgentName = "WhiteSource NPM Plugin"
```

Comment:
same here, I guess we could get rid of the complete file instead.
If I am not mistaken, this is anyway no longer used. We kept it as a safety net during the migration to UA ...